analyses/fusion_filtering/05-QC_putative_onco_fusion_distribution.Rmd

```diff
@@ -116,101 +123,108 @@ putative_driver_annotated_fusions_local<-fusion_calls %>%
-# count number of fusions in putative oncogene annotated fused gene are in more than N (countHistology) histologies
-FusionInNhist<-fusion_calls %>% dplyr::select(FusionName,broad_histology) %>% unique() %>% group_by(FusionName) %>% tally(name="count")
-FusionInNhist<-FusionInNhist[FusionInNhist$count>countHistology,]
-FusionInNhist
+```{r get fusions that are found in more than 4 cancer groups}
+# count number of fusions in putative oncogene annotated fused gene are in more than N (countCancerGroup) cancer groups
+FusionInNcg<-fusion_calls %>% dplyr::select(FusionName,cancer_group) %>% unique() %>% group_by(FusionName) %>% tally(name="count")
 
-# plot broad_histologies that have Fusions that are potential false positives and found in multiple histologies
-multiHistFusion<-fusion_calls %>% dplyr::filter(FusionName %in% FusionInNhist$FusionName) %>%
+FusionInNcg<-FusionInNcg[FusionInNcg$count>countCancerGroup,]
+FusionInNcg
+
+# plot cancer groups that have Fusions that are potential false positives and found in multiple cancer groups
+multiCGFusion<-fusion_calls %>% dplyr::filter(FusionName %in% FusionInNcg$FusionName) %>%
   # we want to remove annots column since there is difference in arriba annotation (uniquely has duplication/translocation/deletion values) and StarFusion annotation which will be counted twice since they are not unique
   dplyr::select(-annots) %>% 
   unique() 
 
-ggplot(multiHistFusion,aes(x=multiHistFusion$FusionName,fill=multiHistFusion$broad_histology))+geom_bar()+theme(axis.text.x = element_text(angle=90))+coord_flip()+ylab("count")+xlab("FusionName(Total)")
-
+ggplot(multiCGFusion,aes(x=multiCGFusion$FusionName,fill=multiCGFusion$cancer_group))+geom_bar()+theme(axis.text.x = element_text(angle=90))+coord_flip()+ylab("count")+xlab("FusionName(Total)")
 
 ```
 
-### Filtering fusions found in more than 4 histologies removes 1183 fusions.  
+### Filtering fusions found in more than 4 broad histologies removes 1183 fusions.  
 
-We can assume these fusions to be potential false calls because it is highly unlikely that fusions of biological importance occur in these highly divese histologies. We (Krutika, Jo Lynne and Payal ) also manually reviewed "Total putative oncogene fusion" plots to look for genes that are found in more than 4 histologies and none seemed to interesting/relevant.
+We can assume these fusions to be potential false calls because it is highly unlikely that fusions of biological importance occur in these highly divese cancer groups. We (Krutika, Jo Lynne and Payal ) also manually reviewed "Total putative oncogene fusion" plots to look for genes that are found in more than 4 broad histology and none seemed to interesting/relevant.
 
 ```{r final filtering }
-putative_driver_fusions<-fusion_calls %>% dplyr::filter(!FusionName %in% FusionInNhist$FusionName) %>%
+# count number of fusions in putative oncogene annotated fused gene are in more than N (countBroadHistology) histology group
```

Comment:
Please see that I did not change any of the other codes that generate graphs - and only changed the steps that does the actual filtering.